regexes/device/mobiles.yml

```diff
@@ -5173,7 +5173,7 @@ Huawei:
       model: 'Honor 9 Lite'
     - regex: 'COL-(?:AL[01]0|TL[01]0|L29)'
       model: 'Honor 10'
-    - regex: 'HRY-(?:AL00|LX[12])|HRY-LX1MEB'
+    - regex: 'HRY-(?:AL00|LX[12])|HRY-(LX1MEB|TL00)'
```

Comment:
why dont you write this `- regex: 'HRY-(?:AL00|LX[12]|LX1MEB|TL00)'`?

Reply:
Yes, I will do so. Done;